<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachGameWnd</class>
 <widget class="QWidget" name="AttachGameWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttachGameWnd</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>挂接</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit_update">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);</string>
           </property>
           <property name="text">
            <string>选择要挂接的游戏</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tableView">
           <property name="styleSheet">
            <string notr="true">QHeaderView::section 
{
	color: black;	
	border-right: 1px solid rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}

QTableCornerButton::section{
	background-color: rgb(255, 255, 255);
}
QTableView
{
	selection-background-color:lightblue;
	selection-color:Black;
	background-color: rgb(255, 255, 255);
}
</string>
           </property>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="checkBox_syncWindow">
             <property name="text">
              <string>窗口同步</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_refresh">
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_attach">
             <property name="text">
              <string>挂接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_unAttach">
             <property name="text">
              <string>卸载</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_login">
             <property name="text">
              <string>登陆</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="AccountForm" name="accountForm">
      <attribute name="title">
       <string>账号</string>
      </attribute>
     </widget>
     <widget class="GameDebugWgt" name="debugWgt">
      <attribute name="title">
       <string>调试</string>
      </attribute>
     </widget>
     <widget class="GameSysCfgWgt" name="sysCfgWgt">
      <attribute name="title">
       <string>配置</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AccountForm</class>
   <extends>QWidget</extends>
   <header>AccountForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameDebugWgt</class>
   <extends>QWidget</extends>
   <header>GameDebugWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameSysCfgWgt</class>
   <extends>QWidget</extends>
   <header>GameSysCfgWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
